Source/Scene/Scene.js

```diff
@@ -1841,6 +1907,10 @@ define([
         var shadowsEnabled = scene.frameState.shadowHints.shadowsEnabled;
         var lightShadowsEnabled = shadowsEnabled && (scene.frameState.shadowHints.lightShadowMaps.length > 0);
 
+        if (scene._logDepthBuffer && defined(command.derivedCommands.logDepth)) {
+            command = command.derivedCommands.logDepth.logDepthCommand;
```

Comment:
I don't know the complete context, but perhaps `logDepth.logDepthCommand` -> `logDepth.command`?